specification/common-types/resource-management/v5/network-security-perimeter.json

```diff
@@ -0,0 +1,275 @@
+              "type": "string"
+            }
+          },
+          "subscriptions": {
+            "type": "array",
+            "description": "Azure subscriptions to match",
+            "items": {
+              "type": "string"
+            }
+          },
+          "networkSecurityPerimeters": {
+            "type": "array",
+            "description": "References the associated network security perimeters",
+            "items": {
+              "$ref": "#/definitions/NetworkSecurityPerimeterReference"
+            }
+          }
+        }
+      },
+      "NetworkSecurityPerimeterReference": {
+        "type": "object",
+        "description": "A reference to an existing network security perimeter (NSP).",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "The Azure resource ID of the network security perimeter."
+          },
+          "perimeterGuid": {
+            "type": "string",
+            "description": "Unique ID of the network security perimeter."
+          },
+          "location": {
+            "type": "string",
+            "description": "The Azure region of the network security perimeter."
+          }
+        }
+      },
+      "NetworkSecurityPerimeterProfile": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "accessRulesVersion": {
+            "type": "string",
+            "description": "Version of the access rules, affecting their interpretation"
+          },
+          "accessRules": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/AccessRule"
+            },
+            "x-ms-identifiers": []
+          }
+        }
+      },
+      "ProvisioningIssue": {
+        "type": "object",
+        "description": "Records an issue which occurred while trying to apply network security perimeter configuration changes",
+        "readonly": true,
+        "properties": {
+          "name": {
+            "type": "string",
+            "readonly": true
+          },
+          "properties": {
+            "$ref": "#/definitions/ProvisioningIssueProperties"
+          }
+        }
+      },
+      "ProvisioningIssueProperties": {
+        "type": "object",
+        "readOnly": true,
+        "description": "The properties of a provisioning issue.",
+        "properties": {
+          "issueType": {
+            "description": "Type of provisioning issue",
+            "type": "string",
+            "readOnly": true
+          },
+          "severity": {
+            "description": "Provisioning state of network security perimeter configuration propagation",
+            "enum": [
+              "Warning",
+              "Error"
+            ],
+            "type": "string",
+            "readOnly": true,
+            "x-ms-enum": {
+              "name": "Severity",
+              "modelAsString": true
+            }
```

Comment:
under   "x-ms-enum": {

you can explain values such as:

```
            "values": [
              {
                "value": "Warning",
                "description": "This can cause connectivity issues after provisioning succeeds."
              },
              {
                "value": "Error",
                "description": "This will cause association provisioning to fail."
              }
            ]
          },
```